Android/notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/FcmTemplateRegistration.java

```diff
@@ -0,0 +1,66 @@
+/*
+Copyright (c) Microsoft Open Technologies, Inc.
+All Rights Reserved
+Apache 2.0 License
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+See the Apache Version 2.0 License for specific language governing permissions and limitations under the License.
+ */
+
+package com.microsoft.windowsazure.messaging;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+/**
+ * Represents FCM template registration
+ */
+public class FcmTemplateRegistration extends TemplateRegistration {
```

Comment:
Need to mark `FcmTemplateRegistration` and `FcmNativeRegistration` classed as deprecated

Reply:
Ok, but maybe GcmTemplateRegistration ?